### DISCUSSION TITLE
Question about the file extraction

### DISCUSSION THREAD

**Discussion by ARHCOS** [zeroruka/GI-Assets#224] [Q&A]
Is it possible to have a step by step guide to learn how to extract the ressources from GI (with textures, json files and 3D model).

**Comment by m4urlclo0** [comment]
you can join this server, in the #faq channel you can find some tutorials (https://discord.com/invite/JAeB5jGdcn)

**Comment by artneem12** [comment]
when will you add Focalors?
Furina's model in black clothes is broken, the texture does not work in .blender
